server/package.json

```diff
@@ -6,15 +6,26 @@
   "scripts": {
```

Comment:
As discussed, you might want to consider consolidating the Server package json with the main package json

Reply:
My latest commit resolves this issue!